### PR TITLE
Update download file and README for ROMFS (related commit :  fa2cd4c )

### PR DESCRIPTION
**Pull request by HONGCHAEHEE**
- Update README and partition_map.cfg to guide about ROMFS and its partition
- Update artik053 download file to remove the temporary cfg file after flashing is done

related commit id : fa2cd4cc75ce3ef66212dfe4b615b0e4250135

### CODE REVIEW

File: build/configs/artik053/artik053_download.sh [hunk 1]
```diff
@@ -73,7 +73,7 @@ main()
 				pushd ${OS_DIR_PATH}
 				sh ../tools/fs/mkromfsimg.sh
 				if [ ! -f "../build/output/bin/romfs.img" ]; then
-					echo "ROMFS image is not present"
+					echo "ROMFS image is not present. Install genromfs."
```
Comment (sunghan-chang) [inline build/configs/artik053/artik053_download.sh]: Why do you add it in here?

Comment (HONGCHAEHEE) [reply build/configs/artik053/artik053_download.sh]: For notice. If genromfs is not installed executing mkromfsimg.sh could be failed.

Comment (sunghan-chang) [reply build/configs/artik053/artik053_download.sh]: I think here is not an appropriately location to say this.
How about move it to romfs [[README]](https://github.com/Samsung/TizenRT/blob/master/tools/fs/README_ROMFS.md) as a pre-condition?

File: build/configs/artik053/README.md [hunk 2]
```diff
@@ -93,11 +93,18 @@ openocd -f artik053.cfg -c ' \
 
 ## ROMFS
 
+### Pre-condition
```
Comment (sunghan-chang) [inline build/configs/artik053/README.md]: No, not here. genromfs is a common tool. There is no dependancy with board. I gave a link to tools/fs/README_ROMFS.md

**Comment by sunghan-chang** [comment]
I mean change a commit message, not pull request title. Without see code change, we should know what you modify thru only commit message.

**Pull request merged**